<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalInfo</class>
 <widget class="QWidget" name="PersonalInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Board" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>400</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="Profile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>300</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
       <item>
        <widget class="QLabel" name="PrimaryNumLabel">
         <property name="text">
          <string>账号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NumLabel">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
       <item>
        <widget class="QLabel" name="UserNameLabel">
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <item>
        <widget class="QLabel" name="SexLabel">
         <property name="text">
          <string>性别：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="SexComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
       <item>
        <widget class="QLabel" name="BirthdayLabel">
         <property name="text">
          <string>生日：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="BirthdayDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
       <item>
        <widget class="QLabel" name="DescLabel">
         <property name="text">
          <string>个性签名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="DescPlainTextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
